src/stache/includes/components/tabs-containers.html

```diff
@@ -0,0 +1,27 @@
+<p>
+  If tabs and vertical tabs contain removable items such as lists or reports, don't create entries in the browser history.
+</p>
+
+<div class="screenshot screenshot-info"> 
+  <img src="~/assets/img/guidelines/back-button/tabcontainers-backbutton.png" alt="Tabs as containers for viewing items" />
+</div>
+
+<br>
+
+<sky-row>
+  <sky-column>
+    <img class="screenshot screenshot-info" src="~/assets/img/guidelines/back-button/verticaltabs-backbutton.png" alt="Vertical tabs as containers for viewing items" />
+  </sky-column>
+</sky-row>
+
+<div class="sky-subsection-heading" style="margin-top: 20px;">Focus</div>
+
+<p>
+  Based on guidance from the <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel">WAI-ARIA Authoring Practices for tabs</a>, maintain focus on activated tabs.
```

Comment:
I'd flip the order of this sentence to get the recommendation up front and the link at the end: "Maintain focus on activated tabs as recommended in the W3C's WAI-ARIA Authoring Practices for tabs."